ui-lib/src/styles/palettes/light.json

```diff
@@ -38,7 +38,8 @@
       "contrast": "hsla(0, 0%, 100%, 1)"
     },
     "grey": {
-      "border": "hsla(0, 0%, 90%, 1)"
+      "border": "hsla(0, 0%, 90%, 1)",
+      "borderDark": " hsla(0, 1%, 23%, 1)"
```

Comment:
```suggestion
```

I think that we must only have one border key with different values depending on the theme.

_light.json_
```json
"grey": {
      "border": "hsla(0, 0%, 90%, 1)",
}
```

_dark.json_
```json
"grey": {
      "border": "hsla(0, 1%, 23%, 1)"
}
```

Thanks to that styles will be reactive.

Reply:
I read the figma has two different colors _per theme_, you are reading it as one per theme, which makes more sense.
I'll sync with design on this

Reply:
Confirmed by design, there's one design per theme, changed it to match